README.md

```diff
@@ -503,6 +504,22 @@ _[Extensible Messaging and Presence Protocol](https://en.wikipedia.org/wiki/XMPP
 - [Salut à Toi](http://www.salut-a-toi.org/) - Multipurpose, multi frontend, libre and decentralized communication tool. ([Source Code](http://repos.goffi.org/sat)) `AGPL-3.0` `Python`
   - [Libervia](http://wiki.goffi.org/wiki/Libervia/en) - Web frontend from Salut à Toi. ([Source Code](http://repos.goffi.org/libervia)) `AGPL-3.0` `Python`
 
+## Community-Supported Agriculture (CSA)
+
+**[`^        back to top        ^`](#)**
+
+_Management and administration tools for community supported agriculture and food cooperatives_
+
+- [ACP Admin](https://acp-admin.ch/) - CSA administration. ([Source Code](https://github.com/acp-admin/acp-admin/)) `MIT` `Ruby`
```

Comment:
Could you elaborate a little more on the functuions of this application in the description.

Reply:
Oh, I thought you meant a description for the whole category.
Item description now added too.